models/validators/MethodValidator.cfc

```diff
@@ -35,6 +35,8 @@ component
 		any validationData,
 		struct rules
 	){
+		var errorMetadata = {};
```

Comment:
I think `errorMetadata` is assuming this will only be used for errors, but in reality I think this can be helpful for ANY type of metadata.  It could be errors, messages, i18n, etc.  So let's try to use the concept of just `metadata`. Thoughts?

Reply:
I always like the idea of giving users more flexibility with `metadata`, however, in this case, the data will always be added to the `ValidationResult` object's `errorMetadata` property - so it might be confusing to some users.

`validationResult.newError( argumentCollection = args ).setErrorMetadata( errorMetadata )`

 Unless you think there should be another way to retrieve the metadata outside of the validation result?